pkg/controller/lifecycle/actuator.go

```diff
@@ -576,12 +574,68 @@ func getSeedResources(oidcReplicas *int32, hibernated bool, namespace, genericKu
 	return resources, nil
 }
 
+func buildHPA(namespace string, k8sVersion *semver.Version) client.Object {
+	if versionutils.ConstraintK8sGreaterEqual123.Check(k8sVersion) {
+		return &autoscalingv2.HorizontalPodAutoscaler{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      constants.ApplicationName,
+				Namespace: namespace,
+			},
+			Spec: autoscalingv2.HorizontalPodAutoscalerSpec{
+				ScaleTargetRef: autoscalingv2.CrossVersionObjectReference{
+					APIVersion: appsv1.SchemeGroupVersion.String(),
+					Kind:       "Deployment",
+					Name:       constants.ApplicationName,
+				},
+				MinReplicas: pointer.Int32(1),
```

Comment:
Nit:
Can you declare min(max)Replicas and targetUtilization as variables at the start of the function so we do not repeat the values in two places. I think that this will be easier to read.